lms/static/sass/discussion/_discussion-v1.scss

```diff
@@ -18,8 +18,11 @@
     box-shadow: none;
     height: 40px;
     text-shadow: none;
+    font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif; // without this, it would fallback to lms button tag style
```

Comment:
1) Separate pieces of the font-family list with spaces
2) Use a variable for fonts so they can be themed. There's a couple places where this exact font stack is already specified, you may be able to pull the variable from there.

```
$> git grep Helvetica *.scss
cms/static/sass/partials/_variables.scss:$f-sans-serif: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
common/static/sass/bourbon/addons/_font-family.scss:$helvetica: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
common/test/test-theme/cms/static/sass/partials/_variables.scss:$f-sans-serif: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
lms/static/sass/partials/base/_variables.scss:$f-sans-serif: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif !default;
lms/static/sass/partials/base/_variables.scss:$header-sans-serif: 'Open Sans', Arial, Helvetica, sans-serif !default;
themes/example/lms/static/sass/base/_variables.scss:$brand-font: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
```